src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -42,6 +42,10 @@ public static IEnumerable<object[]> SerializableEqualityComparers_MemberData()
         /// </summary>
         public static IEnumerable<object[]> SerializableObjects_MemberData()
         {
+            // Save old culture and set a fixed culture for object instantiation
+            CultureInfo oldCulture = CultureInfo.CurrentCulture;
+            CultureInfo.CurrentCulture = new CultureInfo("en-US");
```

Comment:
I would recommend any test setting current culture to be executed remotely even if the test not UAP specific test. This will make the test more predictable and reliable.

by the way, we have a request (and tracking issue) to enable the way in uap to use current culture as it is used in the desktop apps.